app/views/options.html

```diff
@@ -31,9 +31,87 @@
     </label>
     <p>Example: {{10 * options.velocityUnit().mult|longnum}} meat/{{options.velocityUnit().name}}</p>
   </div>
+<div>
+	<button id="login" style="display:none">Log in with Dropbox</button>
+	<a id="logout" href="#" style="display:none">Log out</a>
+	<button id="dbsave" style="display:none">save to Dropbox</button>
+	<button id="dbreset" style="display:none">reset Dropbox</button>
+	<script>
```

Comment:
This needs its own js file. (Most of swarmsim is coffeescript, but mixing some js in is fine.) No `<script>` tags in swarmsim's html files, except the autogenerated ones in index.html - it's too hard to add tests later, inconsistent with code that's already there, and the global namespace gets messy. How to do that with angular:

* Create a controller: `yo angular:controller <name> --coffee=false` if you got yeoman working ([install.sh](https://github.com/erosson/swarm/blob/master/install.sh) should install it for you), otherwise just copy one of the other controllers in `app/scripts/controller` and its include line in index.html.
* `<div ng-controller="<name>">` somewhere to run the code in the controller you created, probably in the line 34 `<div>`.